### PR TITLE
SQL: SELECT 1 not working properly #4067

### PR DESCRIPTION
**Pull request by RawrLion**
added tests for #4067

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/clause/select/Select1Test.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        };
+        assertSqlQueryRows("'SELECT number' query should return one row with that number", expectedRows, sqlQuery);
+    }
+
+    /**
+     * #4067
+     */
+    @Test
+    public void testSelectNumberSum() {
+        String twelve = "12";
```
Comment (GregoryKutuzov) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/select/Select1Test.java]: why not to use integer and toString?

File: src/test/java/com/axibase/tsd/api/method/sql/clause/select/Select1Test.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        String fourty = "40";
+        String sqlQuery = "SELECT " + twelve + "+" + fourty;
```
Comment (vtols) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/select/Select1Test.java]: I also think that these constants are redundant

File: src/test/java/com/axibase/tsd/api/method/sql/clause/select/Select1Test.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package com.axibase.tsd.api.method.sql.clause.select;
+
+import com.axibase.tsd.api.method.sql.SqlTest;
+import org.testng.annotations.Test;
+
+public class Select1Test extends SqlTest {
```
Comment (VeselovAlex) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/select/Select1Test.java]: Wow, such number

File: src/test/java/com/axibase/tsd/api/method/sql/clause/select/Select1Test.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        };
+        assertSqlQueryRows("'SELECT 1' query should return one row with symbol '1'", expectedRows, sqlQuery);
+    }
+
+    /**
+     * #4067
+     */
+    @Test
+    public void testSelectNumber() {
+        String twelve = "12";
```
Comment (VeselovAlex) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/select/Select1Test.java]: inline it

File: src/test/java/com/axibase/tsd/api/method/sql/clause/select/Select1Test.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
+package com.axibase.tsd.api.method.sql.clause.select;
+
+import com.axibase.tsd.api.method.sql.SqlTest;
+import org.testng.annotations.Test;
+
+public class Select1Test extends SqlTest {
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/select/Select1Test.java]: Please, rename class name for better reading.

**Pull request merged**